src/System.Threading.Channels/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      netcoreapp;
```

Comment:
Curious why you added this since the test doesn't differ.

Reply:
I'll remove that.  It's left over from when I actually was adding/testing APIs.